src/opendr/engine/data.py

```diff
@@ -211,15 +211,32 @@ def __str__(self):
 class Image(Data):
     """
     A class used for representing image data.
+    OpenDR uses NCHW/RGB conventions
     This class provides abstract methods for:
     - returning a NumPy compatible representation of data (numpy())
+    - loading an input directly into OpenDR compliant format (open())
+    - getting an image into OpenCV- compliant format (opencv()) for visualization purposes
     """
 
-    def __init__(self, data=None, dtype=np.uint8):
+    def __init__(self, data=None, dtype=np.uint8, guess_format=True):
+        """
+        Image constructor
+        :param: dtype type of the image data provided
+        :guess_format: try to automatically guess the type of input data and convert them to OpenDR format
```

Comment:
This doesn't adhere to the usual way of documentation. It should be something along these lines:

```
:param data: blablabla
:type data: numpy-array-like
:param dtype: ....
:type dtype: data-type
:param guess_format: ...
:type guess_format: bool
```

Reply:
Thanks for noticing. I was going to clean up the code later today or tomorrow.